src/classes/groundplaces.ts

```diff
@@ -334,6 +334,14 @@ export class GroundPlacesController {
     try {
       this.storageService.updatePlace(stopClusterGpuid, propertiesToUpdate, GroundPlaceType.CLUSTER);
 
+      // If stopCluster name doesn't respect this format: CITY, REGION, COUNTRY
+      const stopClusterNameRegex = /^\w+\,\s*\w+\,\s*\w+$/;
+      if (propertiesToUpdate?.name && !stopClusterNameRegex.test(propertiesToUpdate?.name)) {
```

Comment:
```suggestion
      if (propertiesToUpdate?.name && stopClusterNameRegex.split(',').length !== 3) {
```

As you wil not match `Calais, Hauts-de-France, France`which is a valid names because of stresses

Reply:
i changed the regex is it good for you ?

Reply:
The regex stil does't handle stressed letters and `-`